src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react';
+
+interface Props {
+  errorMessage: string;
+  setErrorMessage: (message: string) => void;
+  hideErrorNotice: () => void;
+  errorNoticeRef: React.RefObject<HTMLDivElement>;
+}
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage,
+  hideErrorNotice,
+  errorNoticeRef,
+}) => {
+  const errorNoticeDiv = errorNoticeRef.current as HTMLDivElement;
+
+  const showErrorNotice = () => {
+    errorNoticeDiv.classList.remove('hidden');
+  };
+
+  useEffect(() => {
+    if (errorMessage) {
+      showErrorNotice();
+      setTimeout(() => {
+        hideErrorNotice();
+        setErrorMessage('');
+      }, 3000);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [errorMessage]);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      // eslint-disable-next-line max-len
+      className="notification is-danger is-light has-text-weight-normal hidden"
```

Comment:
Instead of adding/removing classes using DOM API:
![image](https://github.com/user-attachments/assets/ad78a286-5d55-4030-a765-42f8cbed5632)

you should write the correct className based on `!!errorMessage` state

Reply:
Thanks! And duly noted!